From 3f9f7aa864cb1d8e7f2dc14272e9e686b63d0d9d Mon Sep 17 00:00:00 2001
From: SangeethaRao <sangeetha.rao@intel.com>
Date: Fri, 2 Oct 2015 19:18:15 -0500
Subject: [PATCH 64/86] drivers/pci/host: Updated Kconfig

Fixed X9/XLF Build issue caused by inclusion of
ARM32 PCIe driver in addition to ARM64 PCIe driver.

Signed-off-by: SangeethaRao <sangeetha.rao@intel.com>
[Original patch is from linux-yocto-4.1 axxia/base branch.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 drivers/pci/host/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/pci/host/Kconfig b/drivers/pci/host/Kconfig
index d68496e..3484e10 100644
--- a/drivers/pci/host/Kconfig
+++ b/drivers/pci/host/Kconfig
@@ -3,7 +3,7 @@ menu "PCI host controller drivers"
 
 config PCI_AXXIA
 	bool "AXXIA PCIe controller"
-	depends on ARCH_AXXIA && ARM64
+	depends on ARCH_AXXIA && !ARM64
 	default ARCH_AXXIA
 	help
 	 Enables support for the Axxia X7 PCIe controller.
-- 
1.7.5.4

